lib/plugins/create/templates/aws-nodejs/serverless.yml

```diff
@@ -1,96 +1,74 @@
-# Welcome to Serverless!
-#
-# This file is the main config file for your service.
-# It's very minimal at this point and uses default values.
-# You can always add more config options for more control.
-# We've included some commented out config examples here.
-# Just uncomment any of them to get that config option.
-#
-# For full config options, check the docs:
-#    docs.serverless.com
-#
-# Happy Coding!
+## The serverless.yml file is the main config file for your service.
+## To see all available serverless.yml properties see the docs http://bit.ly/2rJJjMV
```

Comment:
> sls.io/xyz

That would be so cool 🙏 

Yes, I understand that. The full URLs are just too long 😞